components/demo/demo-snippet.js

```diff
@@ -211,7 +211,7 @@ class DemoSnippet extends LitElement {
 
 		return lines.join('\n')
 			.replace(/ class=""/g, '') // replace empty class attributes (class="")
-			.replace(/_[^=]*="[^"]*"/, '') // replace private reflected properties (_attr="value")
+			.replace(/_[^=]*="[^"]*"[^\s>]/, '') // replace private reflected properties (_attr, _attr="value", but not target="_blank")
```

Comment:
```suggestion
			.replace(/\s_[^\s\/>"'=]*=(?<q>['"]).*?(?<!\\)\k<q>/gi, '') // replace private reflected properties (_attr, _attr="value", but not target="_blank")
```

- Start with a whitespace, since all attributes must follow a tag name or another attribute
- Then an underscore
- Followed by anything that can be in an attribute name
- Then an `=`
- A quote character
- Anything any number of times, lazily
- A matching closing quote to the open quote that isn't escaped
- All of that globally and case insensitive

Reply:
https://regex101.com/r/BrCoBz/1

Reply:
Thanks Danny. 😄